tom.py

```diff
@@ -86,19 +86,19 @@ def post(self, path, data):
         except:
             return r.headers, r.text
 
-    def trigger(self, prs=None, branch=None, title=None):
+    def trigger(self, prs=None, branch="master", title=None):
         path = self.trigger_url
         params = {}
+        repo_names = ",".join([k.lower() for k in prs])
```

Comment:
The square brackets can be dropped here.

Reply:
Dropping the brackets returns the generator, interesting.

Reply:
Yes. Generators can in general be created using `()` instead of `[]`. And if they are the only argument to a function call, the `()` can be dropped. So:
```
",".join(k.lower() for k in prs)
```
but e.g.
```
next((it for it in items if pred(it)), None)
```